requirements.txt

```diff
@@ -1,31 +1,51 @@
+alabaster==0.7.12
 alembic==1.4.2
 aniso8601==8.0.0
+Babel==2.8.0
 bcrypt==3.1.7
 certifi==2020.6.20
 cffi==1.14.0
 chardet==3.0.4
 click==7.1.2
+colorama==0.4.4
+coverage==5.3
 cryptography==3.2
+docutils==0.16
 Flask==1.1.2
 Flask-Bcrypt==0.7.1
 Flask-Migrate==2.5.3
 Flask-RESTful==0.3.8
 Flask-Script==2.0.6
 Flask-SQLAlchemy==2.4.1
+Flask-Testing==0.8.0
 idna==2.10
+imagesize==1.2.0
 itsdangerous==1.1.0
 Jinja2==2.11.2
 Mako==1.1.2
 MarkupSafe==1.1.1
 marshmallow==3.6.0
+packaging==20.4
 psycopg2==2.8.5
 pycparser==2.20
+Pygments==2.7.2
 PyJWT==1.7.1
+pyparsing==2.4.7
 python-dateutil==2.8.1
 python-editor==1.0.4
 pytz==2020.1
 requests==2.24.0
 six==1.14.0
+snowballstemmer==2.0.0
+Sphinx==3.2.1
```

Comment:
sphinx should not be in `requirements.txt` but a `requirements-dev.txt`. We should check the other dependencies. Everything that is used to test should also be in `-dev.txt`.